spec.html

```diff
@@ -32060,7 +32060,23 @@ <h1>Year Number</h1>
 
```

Comment:
In how far is this commit ("Normative: Use Number arithmetic for constructing dates and times") still normative? Now that the version of ECMA-262 this PR is based on is already clarified to use Number arithmetic.

I guess my doubt would be about the changes to MakeDay. If we consider that currently "underspecified" then maybe it's not normative. (I haven't gone through the new definition with the various examples given in https://github.com/tc39/ecma262/issues/1087, though.)

If the MakeDay changes _are_ normative, on the other hand, then I'd recommend splitting them out of this commit into a separate one, so that it's immediately clear to reviewers what they have to pay the most attention to.